src/ja/common/job.py

```diff
@@ -92,6 +99,11 @@ def uid(self, value: str) -> None:
         @param value The new job UID.
         """
 
+    def label(self) -> str:
```

Comment:
Shouldn't the label also be a property of the job?

Reply:
I mean why is there no label parameter in the __init__ function?

Reply:
Sorry, I just forgot to add it.